static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx

```diff
@@ -331,6 +332,11 @@ export function EventDetailsContent({
           </ErrorBoundary>
         </Fragment>
       )}
+      {issueTypeConfig.metricDetectorTriggered.enabled && (
```

Comment:
i think we don't need this, since we check that the event has a specific property

Reply:
Okay yeah good call, using the data source type now to make the determination